parm/archive/master_gefs.yaml.j2

```diff
@@ -5,10 +5,14 @@
 
 datasets:
 {% filter indent(width=4) %}
-    {% include "gefs_extracted_atmos.yaml.j2" %}
-    {% include "gefs_extracted_ocean.yaml.j2" %}
-    {% include "gefs_extracted_ice.yaml.j2" %}
-    {% include "gefs_extracted_wave.yaml.j2" %}
+    {% if DO_EXTRACTVARS %}
+        {% include "gefs_extracted_atmos.yaml.j2" %}
+        {% include "gefs_extracted_ocean.yaml.j2" %}
+        {% include "gefs_extracted_ice.yaml.j2" %}
+        {% include "gefs_extracted_wave.yaml.j2" %}
+    {% else %}
+        {% include "gefs_arcdir.yaml.j2" %}
+    {% endif %}
 {% endfilter %}
 
```

Comment:
Based on @EricSinsky-NOAA's [comment](https://github.com/NOAA-EMC/global-workflow/pull/3211#issuecomment-2577739246), I think this block can be removed entirely.  Similarly, the `gefs_extractvars*` yaml files be removed entirely.

```suggestion
```